### PR TITLE
Set up icons and replaces existing usage

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

- Set up react-icons
- document usage in web/README.md
- Add real icons for panel-collapse-button and clock
- (unrelated) fixes html lang that was changed

Closes https://linear.app/electricitymaps/issue/ELE-1377/set-up-process-for-icons

### CODE REVIEW

**Review by Kongkille — APPROVED** [review]

File: web/README.md [hunk 1]
```diff
@@ -48,6 +48,14 @@ As an eMap internal team member, you can also run the app connected to productio
...
 See [how to edit world geometries](https://github.com/electricityMaps/electricitymaps-contrib/wiki/Edit-world-geometries).
 
+## Icon usage
+
+We use [react-icons](https://github.com/react-icons/react-icons) and by default the app uses the "Heroicons 2" library with "Outline" variants.
+
+If an icon should be missing, then feel free to look into other libraries to find an icon that matches the style. An important rule here is that we ideally want to use only outlined icons (and no filled/solid icons).
+
+Search for icons here: [https://react-icons.github.io/react-icons/](https://react-icons.github.io/react-icons/)
+
```
Comment (VIKTORVAV99) [inline web/README.md:58]: What do you guys think about moving (some of) this to a CONTRIBUTING.md file once we merge this into "master"?

It has some special treatment in GitHub and will get linked when you make a PR or issue. See [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) for more information.

**Review by tonypls — APPROVED** [review]
Looks good!

File: web/README.md (hunk 1 above)
Comment (Kongkille) [reply web/README.md:58]: That's a great idea! The CONTRIBUTING.MD file could then also link throughout the wiki

**Review by Kongkille — APPROVED** [review]

File: web/README.md (hunk 1 above)
Comment (VIKTORVAV99) [reply web/README.md:58]: I'll make a note of it, so I remember it when we are merging this to the main repo. 😅

**Pull request merged**